apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,133 @@
+<!-- JavaScript code -->
+<script>
+    // Your Svelte script here
+</script>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <script src="https://kit.fontawesome.com/c850f0d795.js" crossorigin="anonymous"></script>
+</head>
+
+<!-- Landing Page - Card -->
+<div class="flex items-center justify-center h-screen">
+
+    <!-- Landing Page - Content -->
+    <div class="w-11/12 h-screen p-10 border border-secondary rounded-md shadow-lg">
+        
+        <!-- Landing Page - Header -->
+        <div class="dropdown dropdown-bottom">
+            <div tabindex="0" role="button" class="btn m-1 text-2xl font-semibold normal-case px-6 rounded-full border-none bg-transparent">My Files
+                <span>
+                    <i class="fa-solid fa-caret-down "></i>
+                </span>
+            </div>
+            <ul class="dropdown-content z-[1] menu p-2 shadow-2xl bg-base-100 rounded-box w-52 border">
+                <li><a href="/"><i class="fa-solid fa-folder"></i> New Folder</a></li>
+                <hr>
+                <li><a href="/"><i class="fa-solid fa-file-arrow-up"></i> File Upload</a></li>
+                <li><a href="/"><i class="fa-solid fa-file-arrow-up"></i> Folder Upload</a></li>
+            </ul>
+        </div>
+
+        <!-- Dropdown buttons for User -->
+        <div>
+
+            <!-- Type Dropdown -->
+            <div class="dropdown dropdown-bottom">
+                <div tabindex="0" role="button" class="btn m-1 font-semibold normal-case px-6 border-white hover:border-white">Classification
+                    <span>
+                        <i class="fa-solid fa-caret-down "></i>
+                    </span>
+                </div>
+                <ul class="dropdown-content z-[1] menu p-2 shadow-2xl bg-base-100 rounded-box w-52 border">
+                    <li><a href="/"><i class="fa-solid fa-folder"></i>Open Class</a></li>
+                    <hr>
+                    <li><a href="/"><i class="fa-solid fa-folder"></i>Confidential Class</a></li>
+                    <hr>
+                    <li><a href="/"><i class="fa-solid fa-folder"></i>Secret Class</a></li>
```

Comment:
Do not inline all the code. Separate all the code into different lines

Reply:
Lines of codes separated, Class removed

Reply:
Separated lines of codes for all pages here as well